src/pages/patientView/PatientViewPage.tsx

```diff
@@ -47,6 +51,7 @@ interface IPatientViewState {
     cnaSegmentData: any;
     mutationData: any;
     hotspotsData?: IHotspotData;
+    cosmicData?: ICosmicData;
```

Comment:
That would create a dependency on `PatientViewPage` in the column formatters which I would like to avoid as much as possible :)